### ISSUE TITLE
Make SFX for Hestmor Growls (All 3 Levels) 

### ISSUE THREAD

**Issue by Pseudo-64** [Team-Awkward-Smile/Epilogue#400]
So there are growls in 3 levels: 

- [ ] Growl 1 is a shorter growl that is weaker in nature.  This growl takes place when Hestmor has three or more hearts. 

Here's the animation for the Growl 1: 

![Image](https://github.com/Team-Awkward-Smile/Epilogue/assets/92845232/22b64422-9857-44dc-991e-d2feeadd0641)

- [ ] Growl 2 is a bit longer growl, lasts a little longer and has more strength to it. This growl takes place when Hestmor has two hearts. 

Here's the animation for Growl 2: 

[Note: Growl 2 Animation pending from mint forwarding me, because I am having trouble downloading it from the spreadsheet, so I will link the spreadsheet for reference here] 

https://docs.google.com/spreadsheets/d/1CpQs6cJEgQ-O6-qxakZzvbrNd6LtvgNXqRC4CnLxuvs/edit?gid=79244261#gid=79244261

![Image](https://github.com/user-attachments/assets/20c4b9c5-a7ee-41ce-9007-9fe1d848f8c5)

- [ ] Growl 3 is the ultimate growl/roar/scream of Hestmor, as this growl takes place when Hestmor's health is down to just one heart and it is not only longest growl, but also there is a bit of desperation in her voice as she basically teetering at the edge of life and death. 

https://docs.google.com/spreadsheets/d/1CpQs6cJEgQ-O6-qxakZzvbrNd6LtvgNXqRC4CnLxuvs/edit?gid=79244261#gid=79244261

![Image](https://github.com/user-attachments/assets/650366fe-ed33-4f0b-b066-cd21d340361f)

(Note for meb: All of these are old notes for Davide, but you are free to read it if you want) 

So this how the main character looks:

![Image](https://github.com/Team-Awkward-Smile/Epilogue/assets/92845232/f500e271-9ea0-4dff-bf85-b8a3b591da1a)

Now, she is kind of a human, cat hybrid. And her voice is on the more heavier side of things. Think of almost a roar of a lioness but soften it a down, but make it more vicious than a scream of a human/cat. 

So basically on a scale of 1-10 imagine: 

1 - being the angry meow of a cat or angry shout of a human. 

10 - being the roar of a lionness. 

 What I need on that scale of 1-10 is a **7**. More inclined towards lionness in how demonic it sounds, but still having elements of a cat/human screaming.

**Comment by Pseudo-64** [comment]
@Mediocre-At-Best I have attached the growl 2 and growl 3 gifs onto the description

**Comment by Mediocre-At-Best** [comment]
[HeartGrowls.zip](https://github.com/user-attachments/files/32093329/HeartGrowls.zip) @Pseudo-64
